API/Controllers/ImageController.cs

```diff
@@ -101,10 +101,38 @@ public async Task<ActionResult> GetSeriesCoverImage(int seriesId, string apiKey)
     {
         var userId = await _unitOfWork.UserRepository.GetUserIdByApiKeyAsync(apiKey);
         if (userId == 0) return BadRequest();
-        var path = Path.Join(_directoryService.CoverImageDirectory, await _unitOfWork.SeriesRepository.GetSeriesCoverImageAsync(seriesId));
-        if (string.IsNullOrEmpty(path) || !_directoryService.FileSystem.File.Exists(path)) return BadRequest(await _localizationService.Translate(userId, "no-cover-image"));
-        var format = _directoryService.FileSystem.Path.GetExtension(path);
+        var seriesMetadata = await _unitOfWork.SeriesRepository.GetSeriesMetadata(seriesId);
+        if (seriesMetadata == null) return BadRequest();
+        string path;
+        switch (seriesMetadata.CoverDisplayOption)
+        {
+            case CoverDisplayOption.Random:
```

Comment:
I'm not a fan of supporting this. I'd need to see a use case to adding it in. I would remove for now.